kubetest/util.go

```diff
@@ -89,10 +91,18 @@ func xmlWrap(name string, f func() error) error {
 		ClassName: "e2e.go",
 		Time:      duration.Seconds(),
 	}
+	if err == nil && !alreadyInterrupted && interrupted {
+		err = fmt.Errorf("kubetest interrupted during step %s", name)
+	}
 	if err != nil {
-		c.Failure = err.Error()
+		if !alreadyInterrupted {
+			c.Failure = err.Error()
```

Comment:
Do not think we want to add anything else here since this is just a wrapper function. In the case where we add an error where one didn't exist because of a timeout then we do (L95)